build.js

```diff
@@ -219,10 +219,35 @@ getCached().then(cached => {
         }
       })
       fs.copyFileSync('./resources/styles.css', `${dir}/styles.css`)
+
+      const manifest = {
+        id: 'app.netlify.stremio-addons',
```

Comment:
Perhaps something like this would be better? There's no affiliation with netlify and it's just the hosting provider that is currently in use but could change anytime in the future.

```suggestion
        id: 'community.stremio.stremio-addons-list',
```